functionary/ui/forms/tasks.py

```diff
@@ -33,18 +35,18 @@ class HTMLDateTimeInput(DateTimeInput):
 
 
 _field_mapping = {
-    "integer": (IntegerField, None),
-    "string": (CharField, None),
-    "text": (CharField, Textarea),
-    "number": (FloatField, None),
-    "file": (CharField, FileInput),
-    "boolean": (BooleanField, None),
-    "date": (DateField, HTMLDateInput),
-    "date-time": (
+    PARAMETER_TYPE.INTEGER: (IntegerField, None),
```

Comment:
I know one of the complaints has been that there are multiple mappings to go update when we add a parameter type.  I don't know that we can get it down to only 1 place, since in a case like this it's a mapping that's hyper specific to the TaskParameterForm.  I think having the constants defined at least makes it a little less error prone.